src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -495,6 +537,7 @@ class CalculatorForm extends React.Component {
         {this.renderEnrolled()}
         {this.renderCalendar()}
         {this.renderKicker()}
+        {__BUILDTYPE__ !== 'production' && this.renderBeneficiaryZIP()}
```

Comment:
This new component kicks of most of the new behavior. Gating it here seems easiest.